loaderconfig.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Having this at the top level seems weird. Would it make more sense inside of `test/fixtures/calc`?